fjord/base/browsers.py

```diff
@@ -120,6 +123,11 @@ def parse_ua(ua):
         platform = 'Linux'
     elif platform.startswith('FreeBSD'):
         platform = 'FreeBSD'
+    elif platform.startswith('CPU iPhone'):
+        # Looks like "CPU iPhone OS 8_3 like Mac OS X" and we want
+        # the "8_3" part.
+        platform_version = platform.split(' ')[3]
```

Comment:
I think you should replace the underscores with periods. iOS versions are generally talked about like `8.3` not `8_3`.